ui/src/containers/Details/Details.test.tsx

```diff
@@ -22,7 +22,9 @@ jest.mock('react-router-dom', () => {
     },
     useParams: () => {
       return {
-        name: 'buildah'
+        name: 'buildah',
+        kind: 'task',
+        catalog: 'tekton'
```

Comment:
can we add a test case which displays `NotFound` page when we pass incorrect params?